frontend/packages/console-shared/src/utils/pod-ring-utils.ts

```diff
@@ -73,6 +74,18 @@ const isPendingPods = (
 ): boolean =>
   (pods?.length === 1 && pods[0].status?.phase === 'Pending') ||
   (!currentPodCount && !!desiredPodCount);
+
+export const getFailedPods = (pods: ExtPodKind[]): number => {
+  let failedPods = 0;
+  pods &&
+    pods.forEach((element) => {
+      if ([AllPodStatus.CrashLoopBackOff, AllPodStatus.Failed].includes(getPodStatus(element))) {
+        failedPods += 1;
+      }
+    });
+  return failedPods;
```

Comment:
```suggestion
  return pods.reduce((acc, currValue) => {
        if ([AllPodStatus.CrashLoopBackOff, AllPodStatus.Failed].includes(getPodStatus(currValue))) {
        acc += 1;
      }
      return acc;
  }, 0);
```

Reply:
updated code to use reduce instead of forEach